### ISSUE TITLE
Design Intent - Accessibility Feedback - IVC Forms, 10-10d

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#73392]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- This all looks good! Looking forward to seeing where you end up at Midpoint Review!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** As you start making UI decisions, the one thing I want to flag how you communicate to users that additional documents will be required, eg. Section 2 Page 5b and Page 6b. When screen reader users are working through a form workflow --- especially something long like this and they're two-and-a-half sections into it --- it's not likely for them to have their screen reader announce all the text on the page. What's more likely is they'll be tabbing through interactive elements and only hearing the text included with each of those elements: buttons, links, form input labels, etc. Any text revealed after the checkboxes or radio buttons is likely to be skipped over as though it isn't there at all.
   
   With that in mind, a few options to explore:
   - The radio button group and checkbox group components have a `hint` property that adds helper text for that set of buttons/boxes. If the document requirement can be summarized in a nice concise hint text, that's a clean and easy way to notify users before they even make their selection.
   - Similarly, the [tile variation](https://design.va.gov/storybook/?path=/docs/uswds-va-radio--tile) for each component supports adding a `description` prop on a per-option basis. You can display different helper text for each option in your set.
   
   If neither of those is workable, then you could also:
   - Include the message as a stand-alone page when users press Continue, without any interactive elements on the page. It would be a bigger interruption to the form flow, but a page with no fields on it (and descriptive headings) should be distinct enough to prompt screen reader users to explore the page more than just tabbing through. I'd recommend this if the message is longer than a couple of lines.
   - Include the message as a message beneath the fieldset, but use `aria-describedby` or similar to ensure it's announced by screen readers when users select the relevant option. This is my least-preferred option, but it can work too.
   
   There are probably some other options too. Just something to be thinking about!
